content/plugins/notes/kubernetes.md

```diff
@@ -0,0 +1,207 @@
+---
+name: kubernetes
+---
+
+You have added the two Kubernetes plugins to your Backstage application:
+
+-   **frontend**: it will take care of displaying the information to the user.
+-   **backend**: it will take care of connecting to the Kubernetes clusters and sending the information to the frontend.
+
+After installing the plugings, you have to configure them in two steps:
+
+1. Allow the backend to collect objects from your Kubernetes cluster(s).
+2. Surfacing your Kubernetes objects in catalog entities
+
+## Configuring Kubernetes Clusters
+
+Here is a complete example of a configuration entry:
+
+```yaml
+# app-config.yaml
+kubernetes:
+    serviceLocatorMethod: 'multiTenant'
+    clusterLocatorMethods:
+        - 'config'
+    clusters:
+        - url: http://127.0.0.1:9999
+          name: minikube
+          authProvider: 'serviceAccount'
+          serviceAccountToken:
+              $env: K8S_MINIKUBE_TOKEN
+        - url: http://127.0.0.2:9999
+          name: gke-cluster-1
+          authProvider: 'google'
+```
+
+Here is the list of fields in detail:
```

Comment:
Let's link to [the official docs](https://backstage.io/docs/features/kubernetes/configuration) instead of duplicating this information please. It's too likely to go stale.